examples/facebook/docs/tutorial-extras/translate-your-site.md

```diff
@@ -39,7 +39,7 @@ Start your site on the French locale:
 npm run start -- --locale fr
 ```
 
-Your localized site is accessible at `http://localhost:3000/fr/` and the `Getting Started` page is translated.
+Your localized site is accessible at [http://localhost:3000/fr/](http://localhost:3000/fr/) and the `Getting Started` page is translated.
```

Comment:
Please don't modify the `examples` folder—these are autogenerated. Otherwise, LGTM.

Reply:
Done,
@Josh-Cena your PR response might be a legit candidate for the fastest PR response in Github's history ;-p